nautobot_netbox_importer/generator/source.py

```diff
@@ -417,6 +418,23 @@ def __str__(self) -> str:
         """Return a string representation of the wrapper."""
         return f"{self.__class__.__name__}<{self.content_type} -> {self.nautobot.content_type}>"
 
+    def cache_record_uids(self, source: RecordData, uid: Optional[Uid] = None) -> Uid:
+        """Cache record identifier mappings."""
+        if not uid:
+            return self.get_pk_from_data(source)
```

Comment:
Would you mind updating the docstring to clarify the behavior? It feels non-obvious to me that calling `cache_record_uids(source, uid)` updates the cache, but `cache_record_uids(source, None)` does nothing at all to the cache but *does* derive and return a uid?

Reply:
Updated docstrings and renamed variable for readability.